Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,510 @@
+The problem found with this was the required velocity change.
+
+This project will also be great for
+the members to put on their resume. At the end of this project a detailed document will be made which outlines the most important aspects of the project.
+Then the RCS will build off the momentum created by this project. And the understanding of space propulsion inside SPEX will continue to grow and flourish.
+
+Once the CGT is complete, the team can move onto the next phase of space propulsion research. This is the RCS\@. Reaction control systems are used very
+frequently in space propulsion and attitude adjustment systems as well as rockets. Building an RCS is difficult. But if the team
+has prior experience with cold gas propulsion, their job becomes significantly easier. The team can refer back to their data for this.
+
+After the culmination of the cold gas thruster project, the team can try to design, fabricate, and test an aerospike nozzle, only if time permits.
+There are benefits to aerospikes but there are also disadvantages. The advantages are concerned with the region of optimal expansion. The aerospike nozzle can match the
+ambient pressure over a larger region and therefore can produce more thrust. However, the disadvantages are increased thermal loading, which is not terribly important here but is worht mentioning,
+and high design and fabrication complexity.
+
+\section*{Acknowledgements}
+The author would like to thank Dr. Garrick and Dr. Olles for their help in understanding the theory of nozzles and the review of the system before testing.
+
+
+\onecolumn
+\appendices{}
+\section{\textbf{Detailed Fall Timeline}}
+\centering
+\begin{tabular}{ |p{1.25cm}||p{11.0cm}| }
+  \hline
+  Week \# & Plan \\
+  \hline
+  1 & Introductions \& Fall plan discussion \\
+  \hline
+  2 & Beginner Project: Supersonic Nozzle Design  \\
+  & Design Phase: Test stand \\
+  \hline
+  3 & Beginner Project: Nozzle Printing and Testing on weekend \\
+  & Design Phase: Propulsion System \\
+  \hline
+  4 & Design Phase: Propulsion System \\
+  \hline
+  5 & Design Phase: Arduino and MATLAB design \\
+  \hline
+  6 & Design Phase: Arduino and MATLAB design \\
+  \hline
+  7 & Fabrication Phase: Material order and making Solidworks drawings\\
+  \hline
+  8 & Fabrication Phase: Machining \\
+  \hline
+  9 & Fabrication Phase: Machining and assembly \\
+  \hline
+  10 & Experimentation Phase: Test 1 \\
+  \hline
+  11 & Experimentation Phase: Test 2 \\
+  \hline
+  12 & Data Analysis Phase: Data Organization \\
+  \hline
+  13 & Data Analysis Phase: Noise and error analysis \\
+  \hline
+  14 & Data Analysis Phase: Performance Analysis \\
+  \hline
+  15 & Project Culmination Report writing \\
+  \hline
+  16 & Project Culmination Report finalization \\
+  \hline
+\end{tabular}
+
+\newpage
+\section{\textbf{Cold Gas Thruster Preliminary Bill of Materials}}
+
+\centering
+\begin{tabular}{ |p{4cm}||p{1.0cm}|p{1.5cm}|p{2.5cm}|p{2.0cm}|p{2.0cm}| }
```

Comment:
I should make a tables tutorial...

Your text is not centered because of how you have structured the arguments here: 
```tex
\begin{tabular}{these arguments}
```
These arguments tell LaTeX how to format your columns. What you have here tells latex to make a vertical line, then 4cm wide column, vertical line, then 1cm wide column, etc. Generally, we want LaTeX to do the work for us.
```tex
\begin{tabular}{r c l}
```
This says to make the first column right aligned, middle column center aligned, 3rd column left aligned, and to size the cells however best fits the data.

I strongly oppose vertical lines, and even horizontal lines unless separating row headers from the table body. Tables can be structured to present data clearly without unnecessary lines. I linked to a style guide that shows you how to make tables that are easy to read and look good. It includes sample code to use too. Here it is again: https://www.inf.ethz.ch/personal/markusp/teaching/guides/guide-tables.pdf